Solutions/Infoblox NIOS/Workbooks/Infoblox-Workbook-V2.json

```diff
@@ -1,561 +1,484 @@
+          "value": "DNSTab"
+        },
+        "customWidth": "50",
+        "name": "query - 7",
+        "styleSettings": {
+          "margin": "50"
+        }
+      },
+      {
+        "type": 3,
+        "content": {
+          "version": "KqlItem/1.0",
+          "query": "Infoblox_dnsclient \r\n| where ProcessName == \"named\" and Log_Type == \"client\"\r\n| where Computer in ({InfobloxDevice}) or '*' in ({InfobloxDevice})\r\n| where isempty(ResponseCode) and isnotempty(QueryDomainName)\r\n| summarize count() by tostring(QueryDomainName)\r\n| top 10 by count_",
+          "size": 0,
+          "title": "Top 10 DNS Request Domains",
+          "timeContext": {
+            "durationMs": 604800000
           },
-          "name": "query - 7 - Copy",
-          "styleSettings": {
-            "margin": "50"
+          "queryType": 0,
+          "resourceType": "microsoft.operationalinsights/workspaces",
+          "gridSettings": {
+            "formatters": [
+              {
+                "columnMatch": "count_",
+                "formatter": 3,
+                "formatOptions": {
+                  "palette": "turquoise"
+                }
+              }
+            ],
+            "labelSettings": [
+              {
+                "columnId": "QueryDomainName",
+                "label": "Domain Name"
+              },
+              {
+                "columnId": "count_",
+                "label": "Total"
+              }
+            ]
           }
+        },
+        "conditionalVisibility": {
+          "parameterName": "selectedTab",
+          "comparison": "isEqualTo",
+          "value": "DNSTab"
+        },
+        "customWidth": "50",
+        "name": "query - 7 - Copy",
+        "styleSettings": {
+          "margin": "50"
         }
-      ],
-      "isLocked": false,
-      "fallbackResourceIds": [
-        
-      ],
-      "fromTemplateId": "sentinel-UserWorkbook"
-    }
-  },
-  "resources": [
-    {
-      "name": "[parameters('workbookId')]",
-      "type": "microsoft.insights/workbooks",
-      "location": "[resourceGroup().location]",
-      "apiVersion": "2021-03-08",
-      "dependsOn": [],
-      "kind": "shared",
-      "properties": {
-        "displayName": "[parameters('workbookDisplayName')]",
-        "serializedData": "[string(variables('workbookContent'))]",
-        "version": "1.0",
-        "sourceId": "[parameters('workbookSourceId')]",
-        "category": "[parameters('workbookType')]"
       }
-    }
-  ],
-  "outputs": {
-    "workbookId": {
-      "type": "string",
-      "value": "[resourceId( 'microsoft.insights/workbooks', parameters('workbookId'))]"
-    }
-  },
-  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#"
-}
\ No newline at end of file
+    ],
+    "fallbackResourceIds": [
+      "/subscriptions/4383ac89-7cd1-48c1-8061-b0b3c5ccfd97/resourcegroups/cattesting/providers/microsoft.operationalinsights/workspaces/cat-loganalytics"
```

Comment:
@rushriva A workbook template must not contain any references to resources. Because of that validation are failing now.